ui/app/AppLayouts/Wallet/popups/SignTransactionModalBase.qml

```diff
@@ -14,13 +14,15 @@ import StatusQ.Popups 0.1
 import StatusQ.Popups.Dialog 0.1
 
 import shared.controls 1.0
+import shared.stores 1.0
 
 import utils 1.0
 
 StatusDialog {
     id: root
 
     required property int loginType // RootStore.loginType -> Constants.LoginType enum
+    required property CurrenciesStore currencyStore
```

Comment:
Please avoid depending on stores when it's not really necessary. Here we need only formatting function and it should be taken via the API.

Ideally we should define that property with formatting function directly in particular subclasses as they have different formatting needs. For one its just `fn(amount, symbol)`, for another it's `fn(amount, symbol, noSymbol)`. Then we take what's really needed. But for simplicity one property in the base in format `fn(amount, symbol, noSymbol)` would be ok as well imo.

Reply:
I just moved this dependency down since all those (similar) popups use the same function internally; here it really makes sense

Reply:
See my further comments below

Reply:
Done